image/cli/mascli/functions/gitops_suite_app_config

```diff
@@ -37,6 +37,8 @@ Maximo Application Suite:
 Maximo Manage:
   --mas-app-settings-server-bundles-size ${COLOR_YELLOW}MAS_APP_SETTINGS_SERVER_BUNDLES_SIZE${TEXT_RESET}   The Maximo Manage Serer Bundle size (snojms, jms, dev or small)
   --mas-app-settings-jms-queue-mount-path ${COLOR_YELLOW}MAS_APP_SETTINGS_JMS_QUEUE_MOUNT_PATH${TEXT_RESET} JMS queue mount path
+  --mas-app-server-bundles-combined-add-server-config-yaml ${COLOR_YELLOW}MAS_APP_SERVER_BUNDLES_COMBINED_ADD_SERVER_CONFIG_YAML${TEXT_RESET}  Yaml file location containing Combined additional Server Configuration for server bundles
```

Comment:
The yaml is expected to have the server configs (xml) to be base64 encoded. Can we add that to this line somehow.